buf/registry/module/v1/commit_service.proto

```diff
@@ -54,6 +54,13 @@ message GetCommitsResponse {
   repeated Commit commits = 1 [(buf.validate.field).repeated.min_items = 1];
 }
 
+// GetCommitsErrorDetails is the error details for the GetCommits RPC.
+message GetCommitsErrorDetails {
+  // This contains the resource_refs that the server is unable to find. This is populated
+  // when a non-found error is returned by the server.
+  repeated ResourceRef not_found_resource_refs = 1;
```

Comment:
`ResourceRef` is `request_only`, should we expand `request_only` to allow error details?